speid/views.py

```diff
@@ -20,26 +21,30 @@ def health_check():
 
 @app.route('/orden_events', methods=['POST'])
 def create_orden_events():
-    if "id" not in request.json or int(request.json["id"]) <= 0:
-        return make_response(jsonify(request.json), 400)
+    try:
+        if "id" not in request.json or int(request.json["id"]) <= 0:
+            return make_response(jsonify(request.json), 400)
 
-    request_id = request.json['id']
-    transaction = (db.session.query(Transaction).
-                   filter_by(orden_id=request_id).one())
+        request_id = request.json['id']
+        transaction = (db.session.query(Transaction).
+                       filter_by(orden_id=request_id).one())
 
-    transaction.estado = Estado.get_state_from_stp(request.json["Estado"])
-    event = Event(
-        transaction_id=transaction.id,
-        type=State.received,
-        meta=str(request.json)
-    )
+        transaction.estado = Estado.get_state_from_stp(request.json["Estado"])
+        event = Event(
+            transaction_id=transaction.id,
+            type=State.received,
+            meta=str(request.json)
+        )
 
-    callback_helper.set_status_transaction(
-        transaction.speid_id,
-        dict(estado=transaction.estado.value))
-    db.session.add(transaction)
-    db.session.add(event)
-    db.session.commit()
+        callback_helper.set_status_transaction(
+            transaction.speid_id,
+            dict(estado=transaction.estado.value))
+        db.session.add(transaction)
+        db.session.add(event)
+        db.session.commit()
+    except Exception as exc:
+        capture_exception(exc)
+        return make_response(jsonify(request.json), 400)
```

Comment:
La respuesta si ocurre una excepción debería de ser `"got it!"` independientemente si ocurrió una excepción. Para que STP no nos reenvié la misma información 20 veces